### ISSUE TITLE
which model to report

### ISSUE THREAD

**Issue by mrezaei92** [fanglinpu/JGR-P2O#5]
Hi,
Thanks for sharing your code.
What model checkpoint do you use to report the results in your paper?
Do you use a separate Validation set (split off from the training set) to find the best model?
Or do you use the test set to find the best model?
